src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -922,24 +922,24 @@ Please download %2$s from the website."</string>
     <string name="revanced_disable_rolling_number_animations_summary_off">Rolling numbers are animated.</string>
-    <string name="revanced_hide_chapters_section_title">Hide chapters sections</string>
-    <string name="revanced_hide_chapters_section_summary_on">Chapters sections are hidden.</string>
-    <string name="revanced_hide_chapters_section_summary_off">Chapters sections are shown.</string>
-    <string name="revanced_hide_info_cards_section_title">Hide info cards sections</string>
-    <string name="revanced_hide_info_cards_section_summary_on">Info cards sections are hidden.</string>
-    <string name="revanced_hide_info_cards_section_summary_off">Info cards sections are shown.</string>
-    <string name="revanced_hide_podcast_section_title">Hide podcast sections</string>
-    <string name="revanced_hide_podcast_section_summary_on">Podcast sections are hidden.</string>
-    <string name="revanced_hide_podcast_section_summary_off">Podcast sections are shown.</string>
+    <string name="revanced_hide_chapters_section_title">Hide Chapters section</string>
+    <string name="revanced_hide_chapters_section_summary_on">Chapters section is hidden.</string>
+    <string name="revanced_hide_chapters_section_summary_off">Chapters section is shown.</string>
+    <string name="revanced_hide_info_cards_section_title">Hide info cards section</string>
+    <string name="revanced_hide_info_cards_section_summary_on">Info cards section is hidden.</string>
+    <string name="revanced_hide_info_cards_section_summary_off">Info cards section is shown.</string>
+    <string name="revanced_hide_podcast_section_title">Hide Explore the podcast section</string>
+    <string name="revanced_hide_podcast_section_summary_on">Explore the podcast section is hidden.</string>
+    <string name="revanced_hide_podcast_section_summary_off">Explore the podcast section is shown.</string>
     <string name="revanced_hide_shopping_links_title">Hide shopping links</string>
     <string name="revanced_hide_shopping_links_summary_on">Shopping links are hidden.</string>
     <string name="revanced_hide_shopping_links_summary_off">Shopping links are shown.</string>
-    <string name="revanced_hide_suggestions_section_title">Hide suggestions sections</string>
-    <string name="revanced_hide_suggestions_section_summary_on">Game sections, music sections, and place sections are hidden.</string>
-    <string name="revanced_hide_suggestions_section_summary_off">Game sections, music sections, and place sections are shown.</string>
-    <string name="revanced_hide_transcript_section_title">Hide transcript sections</string>
-    <string name="revanced_hide_transcript_section_summary_on">Transcript sections are hidden.</string>
-    <string name="revanced_hide_transcript_section_summary_off">Transcript sections are shown.</string>
+    <string name="revanced_hide_suggestions_section_title">Hide attributes sections</string>
```

Comment:
Shouldn't it be attributes section as a video will only have any one of the section and not more than that?

Reply:
Ah, I didn't realize that. Thanks